apps/gateway/test/champions_test.exs

```diff
@@ -295,6 +298,121 @@ defmodule Gateway.Test.Champions do
     end
   end
 
+  describe "campaigns" do
+    test "get campaigns and levels", %{socket_tester: socket_tester} do
+      # Register user
+      {:ok, user} = Users.register("campaign_user")
+
+      # GetCampaigns
+      SocketTester.get_campaigns(socket_tester, user.id)
+      fetch_last_message(socket_tester)
+
+      assert_receive %WebSocketResponse{
```

Comment:
Do you think this approach is better than using `get_last_message()`?

Reply:
I saw the way you used it in previous tests and thought that the intention was clearer using `assert_receive`, do you see any downside to it?

Reply:
Well the one downside I see here is that we have no timeout to the assertion, but that can be fixed two ways:
a. add the timeout parameter to assert_receive which is nil by default
b. use get_last_message() which already has a timeout

I'm not sure why I created get_last_message if just asserting the response works fine. Thing is, we should probably choose one strategy and stick with it, so we should either do all assert_recieve or all get_last_message().

Reply:
As we talked recently, I removed the get_last_message() function and just kept the `assert_receive` with their corresponding matchings.